MIG-1622: Add new {MTC-first} attribute (#84287)
openshift/openshift-docs: modules/migration-about-configuring-proxies.adoc

[id="migration-about-configuring-proxies_{context}"]
= Proxy configuration

> For {product-title} 4.1 and earlier versions, you must configure proxies in the `MigrationController` custom resource (CR) manifest after you install the {mtc-first} Operator because these versions do not support a cluster-wide `proxy` object.

Reviewer: ```suggestion
For {product-title} 4.1 and earlier versions, you must configure proxies in the `MigrationController` custom resource (CR) manifest after you install the {mtc-full} Operator {mtc-short} because these versions do not support a cluster-wide `proxy` object.
```
I think having the (MTC) before Operator would look a bit odd.

> For {product-title} 4.1 and earlier versions, you must configure proxies in the `MigrationController` custom resource (CR) manifest after you install the {mtc-first} Operator because these versions do not support a cluster-wide `proxy` object.

Author: ```suggestion
For {product-title} 4.1 and earlier versions, you must configure proxies in the `MigrationController` custom resource (CR) manifest after you install the {mtc-short} Operator because these versions do not support a cluster-wide `proxy` object.
```

> For {product-title} 4.1 and earlier versions, you must configure proxies in the `MigrationController` custom resource (CR) manifest after you install the {mtc-short} Operator because these versions do not support a cluster-wide `proxy` object.

Reviewer 2: First mention in the module.
```suggestion
For {product-title} 4.1 and earlier versions, you must configure proxies in the `MigrationController` custom resource (CR) manifest after you install the {mtc-first} Operator because these versions do not support a cluster-wide `proxy` object.
```

> For {product-title} 4.1 and earlier versions, you must configure proxies in the `MigrationController` custom resource (CR) manifest after you install the {mtc-first} Operator because these versions do not support a cluster-wide `proxy` object.

Author: ```suggestion
For {product-title} 4.1 and earlier versions, you must configure proxies in the `MigrationController` custom resource (CR) manifest after you install the {mtc-full} Operator because these versions do not support a cluster-wide `proxy` object.
```
